pkgs/development/python-modules/deepwave/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytorch
+, ninja
+, scipy
+, which
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "deepwave";
+  version = "0.0.11";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "ar4";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-d4EahmzHACHaeKoNZy63OKwWZdlHbUydrbr4fD43X8s=";
+  };
+
+  # unable to find ninja although it is available
+  postPatch = ''
+    substituteInPlace setup.cfg --replace "ninja" ""
+  '';
+
+  # The source files are compiled at runtime and cached at the
+  # $HOME/.cache folder, so for the check phase it is needed to
+  # have a temporary home
```

Comment:
Do you mean to complement this one, @Mindavi ?

thanks for the review, and prompt responses!

Reply:
yeah, mention the ninja things here as well and then I think it's good 🙂

Reply:
oki doki, I will fix this today.

Reply:
Done!